guides/common/modules/proc_registering-hosts.adoc

```diff
@@ -91,6 +91,19 @@ If set to `Yes`, public SSH keys will be installed on the registered host.
 The inherited value is based on the `host_registration_remote_execution` parameter.
 It can be inherited, for example from a host group, an operating system, or an organization.
 When overridden, the selected value will be stored on host parameter level.
+ifdef::katello[]
+. From the *REX pull mode* list, select whether you want to deploy {Project} remote execution pull client or not.
++
+If set to `Yes`, {Project} remote execution pull client will be installed on the registered host.
+The inherited value is based on the `host_registration_remote_execution_pull` parameter.
+It can be inherited, for example from a host group, an operating system, or an organization.
+When overridden, the selected value will be stored on host parameter level.
+
+The registered host has to have access to a repository containing `foreman_ygg_worker` package.
+ifndef::satellite[]
```

Comment:
Can you explain to me the logic of embedding this bit inside the katello bit? 
Is this with the view that satellite will ultimately be added to the first ifdef in time?

Reply:
I will test the builds locally to see if this will work as I am unsure.

Reply:
If I build for Satellite, I do not see this appear. If all of the information apart from the client repos is also relevant for Satellite, which it appears to be, I suggest adding `,satellite` after ifdef::katello and then this bit is perfect as it excludes for Satellite. 
```suggestion
ifndef::satellite[]
```